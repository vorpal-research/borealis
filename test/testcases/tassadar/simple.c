
int borealis_nondet();

int main() {
	int a[220] = {0};

	a[0] = 42;
	a[2] = 1;
	a[3] = 2;

	a[221] = 4;

	return a[borealis_nondet()];
}