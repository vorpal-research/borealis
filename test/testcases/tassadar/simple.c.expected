[
"Memory write violation at 0x100374"
]
